packages/experiments/README.md

```diff
@@ -0,0 +1,130 @@
+<p align="center">
+  <img
+    class="project-logo"
+    src="./NCC-1701-a-blue.svg#gh-light-mode-only"
+    alt="WarpDrive"
+    width="120px"
+    title="WarpDrive" />
+  <img
+    class="project-logo"
+    src="./NCC-1701-a.svg#gh-dark-mode-only"
+    alt="WarpDrive"
+    width="120px"
+    title="WarpDrive" />
+</p>
+
+<h3 align="center">Experiments that may or may not make it into the project core</h3>
+<p align="center">Ideal for Having Fun</p>
+<p align="center">SharedWorker + IndexedDB robust request deduplication and replay</p>
+
+> ⚠️ ***Experimental*** ⚠️
+
+- :electron: Dedupe requests across multiple tabs and windows
+- ♻️ Replay requests reliably in any order and still get the latest state of all associated resources
+- 📶 Load new tabs or windows without ever hitting network
+- 💪 Control the Cache lifetime with confidence
+
+
+## Installation
+
+Install using your javascript package manager of choice. For instance with [pnpm](https://pnpm.io/)
+
+```no-highlight
+pnpm add @warp-driveexperiments
+```
+
+## 🚀 Setup
+
+Using the Data Worker requires a small bit of configuration
+
+1. [Configuring the Worker]()
+2. [Configuring the Worker Build]()
+3. [Configuring your App]()
+
+### 1. Configuring the Worker
+
+In `<project>/workers/ember-data-cache-worker.js`
+
+```ts
+import RequestManager from '@ember-data/request';
+import Fetch from '@ember-data/request/fetch';
+import { LifetimesService } from '@ember-data/request-utils';
+import { DataWorker, CacheHandler } from '@warp-drive/experiments';
+import DataStore from '@ember-data/store';
+
+const CONFIG = {
+  apiCacheHardExpires: 120_000, // 2 minutes
+  apiCacheSoftExpires: 30_000, // 30 seconds
+};
+
+class Store extends DataStore {
+  constructor(args) {
+    super(args);
+
+    const manager = (this.requestManager = new RequestManager());
+    manager.use([Fetch]);
+
+    // this CacheHandler differs from the Store's in that it does not
+    // instantiate records for the response. It insteads takes the
+    // ResponseDocument from the cache and caches the request, document
+    // and resource data involved into indexeddb before returning the
+    // original raw response
+    manager.useCache(CacheHandler);
+
+    // our indexeddb cache will respect lifetimes, so registering
+    // a lifetimes service (even if not this one) is important!
+    this.lifetimes = new LifetimesService(this, CONFIG);
```

Comment:
```suggestion
    this.lifetimes = new CachePolicy(CONFIG);
```

@runspired should we also rename `this.lifetimes` property ? I think it left untouched in #9444 
also we don't need to pass `this` to constructor, right ?